jenkins/cleanup_python_code.py

```diff
@@ -44,14 +44,21 @@
     default=None
 )
 def main(sha, repo_root, user_reviewers, team_reviewers, packages, scripts):
-    scripts = scripts.split(',')
-    scripts = "\n".join(scripts)
-    pr_message = "This pull request was generated by the cleanup-python-code Jenkins job, which ran \n{0}\n" \
-                 "The following packages were installed: {1}.".format(scripts, packages)
+    scripts = "\n".join(scripts.split(','))
+    packages = "\n".join(packages.split(','))
+    commit_message = (
+        "Python code cleanup by the cleanup-python-code Jenkins job.\n" +
+        "\n" +
+        "This pull request was generated by the cleanup-python-code Jenkins job, which ran\n" +
+        "{0}\n" +
+        "\n" +
+        "The following packages were installed:\n" +
+        "{1}"
+    ).format(scripts, packages)
 
     pull_request_creator = PullRequestCreator(sha=sha, repo_root=repo_root, branch_name=CODE_CLEANUP_BRANCH_NAME,
                                               user_reviewers=user_reviewers, team_reviewers=team_reviewers,
-                                              commit_message=pr_message, pr_title="Python Code Cleanup",
+                                              commit_message=commit_message, pr_title="Python Code Cleanup",
                                               pr_body=pr_message)
```

Comment:
maybe we should also pass the same commit message for `pr_body` as well, since `pr_message` no longer exists?

Reply:
oops :)

Reply:
Fixed.